src/eval/execute.rs

```diff
@@ -271,4 +310,20 @@ mod tests {
             assert_eq!(result, (header, body));
         }
     }
+
+    #[test]
+    fn test_paste_to_pastebin() {
+        let client = crate::build_client();
+
+        tokio::runtime::Builder::new_current_thread()
+            .enable_all()
+            .build()
+            .unwrap()
+            .block_on(async move {
+                let example_code = "fn main(){}";
+                assert!(paste_to_pb(&client, &example_code)
+                    .await
+                    .is_ok_and(|resp| resp.starts_with(DPASTE_BASE_URL)))
+            })
+    }
```

Comment:
Actually tokio provides [`tokio::test`](https://docs.rs/tokio/latest/tokio/attr.test.html) for testing async code. You don't need to write the initialization code manually.